README.md

```diff
@@ -358,6 +360,12 @@ configure these settings, stay in the **Checkout** section of the **Settings** m
 > Shopify's admin UI.
 > [Follow Shopify's instructions for setting up test payments in your store](https://help.shopify.com/en/manual/payments/shopify-payments/testing-shopify-payments).
 
+Shopify's support for allowing headless checkout on password protected stores has proven erratic, so it is **not**
+recommended that you password protect your Shopify store when using Penny. Instead you should update your Shopify theme
+to redirect users to your Penny site and set up a `robots.txt` file to prevent indexing of your Shopify native store.
+
+The [shopify-headless-theme](https://github.com/instantcommerce/shopify-headless-theme) is an excellent option.
```

Comment:
This is not new, but Shopify has broken checkouts on stores that are password protected. These are the new recommendations for a functional headless store.